Readme.md

```diff
@@ -4,27 +4,34 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# Grid for Blazor - Customize the appearance of elements including row and cell values based on custom conditions
+# Grid for Blazor - Customize cell appearance based on custom conditions
 
-The [Blazor Grid](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid) allows you to customize the appearance of individual UI elements based on custom conditions (conditional formatting).
+The example demonstrates how to customize appearance of the [DevExpress Blazor Grid](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid) elements based on custom conditions.

```

Comment:
DevExpress Blazor Grid elements is a plural form, which renders the article optional. And if the article is optional, we omit it according to our editorial policy.

```suggestion
The example demonstrates how to customize appearance of [DevExpress Blazor Grid](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid) elements based on custom conditions.

```